src/segments/typing.py

```diff
@@ -485,11 +478,14 @@ class CheckboxTaskAttribute(BaseModel):
     default_value: Optional[bool] = None
 
 
-TaskAttribute = Union[
-    SelectTaskAttribute,
-    TextTaskAttribute,
-    NumberTaskAttribute,
-    CheckboxTaskAttribute,
+TaskAttribute = Annotated[
```

Comment:
Yes, this gives extra information to pydantic that the type of this union can be decided/discriminated based on the input_type field.

Reply:
I was looking at this first as a fix for the title problem but it's more complex because the discriminator is nested in a list for the pointcloud sequence annotation.

Reply:
Ok, but so is it good/necessary to have it here or can it be dropped?

Reply:
Ok got it, let's keep it then.